pcdsdevices/mpod_apalis.py

```diff
@@ -246,6 +292,50 @@ def set_current_ramp_speed(self, ramp_speed):
         """
         self.current_ramp_speed.put(ramp_speed)
 
+    @property
+    def polarity(self) -> Polarity:
+        model_str = self.model.get()
```

Comment:
Is it possible to store this via monitor callback and use the cached value instead of getting io in a property?